a/ajaxorg-ace/build_info.json

```diff
@@ -1,11 +1,11 @@
 {
-   "maintainer": "Shubham.Garud@ibm.com",
-   "package_name": "ajaxorg-ace",
-   "github_url": "https://github.com/ajaxorg/ace",
-   "version": "v1.32.3",
-   "default_branch": "master",
-   "build_script": "ajaxorg-ace_ubi_8.7.sh",
-   "package_dir": "a/ajaxorg-ace",
-   "docker_build": false,
-   "validate_build_script": true
+  "maintainer": "shubham-garud",
+  "package_name": "ace",
```

Comment:
@mohitp333 why we changed package name?

Reply:
Hi @nikhil-kalbande it was done by mistake. Please check now.